public_suffix_list.dat

```diff
@@ -982,8 +982,19 @@ fi
 // TODO: Check for updates (expected to be phased out around Q1/2009)
 aland.fi
 
-// fj : https://en.wikipedia.org/wiki/.fj
-*.fj
+// fj : http://domains.fj/
+// Submitted by registry <garth.miller@cocca.org.nz> 2020-02-11
+fj
```

Comment:
The sorting needs adjustment.
https://github.com/publicsuffix/list/wiki/Format

For instance, `ac.fj` needs to be before `com.fj`